docs/modules/ROOT/partials/version-history.adoc

```diff
@@ -119,3 +119,7 @@ _This section is non-normative._
 
 * Prepare 1.0-draft version.
 * Remove discussion of W3C VCs. (This section will be restored in a post-1.0 version.)
+
+*Pending merge*
```

Comment:
Why leave a "pending merge" in the history?

Reply:
Per discussion in meeting today, this is a note-to-self to add an entry in version history. Now that this is to be merged, I'm adjusting to reflect today's date.